.travis.yml

```diff
@@ -0,0 +1,17 @@
+sudo: required
+
+language: python
+
+python:
+  - "2.6"
+  - "2.7"
+  - "3.2"
+  - "3.3"
+  - "3.4"
+  - "3.5"
+  - "3.6"
```

Comment:
We probably also don't need to support every python3 version, but thats open to discussion.